### ISSUE TITLE
Make current use residential go back to blue when "remove" is clicked (trigger refresh)

### ISSUE THREAD

**Issue by edwardchalstrey1** [colouring-cities/colouring-london#251]

**Comment by mdsimpson42** [comment]
I'm assuming this refers to the Land Use category. There are separate colours on the key for "Residential (unverified)" (blue) and "Residential (verified)" (purple). 

Presumably, this should change automatically to purple when at least one person clicks "verify" and back to blue if they click "remove" on the verification?

This doesn't appear to have been done....

![image](https://github.com/colouring-cities/colouring-britain/assets/21125422/c68ab0c0-656e-4a42-825d-2855e717f15e)

**Comment by polly64** [comment]
@mdsimpson42 yes that's right although you shouldn't;t be able to remove someone else's verification only yours
